packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -436,6 +519,55 @@ contract L2Genesis is Script, Artifacts {
         vm.ffi(commands);
     }
 
+    function _overwriteFile(string memory _target, string memory _source) internal {
+        string[] memory updateGenesisOutFileCommands = new string[](3);
+        updateGenesisOutFileCommands[0] = "bash";
+        updateGenesisOutFileCommands[1] = "-c";
+        updateGenesisOutFileCommands[2] = string.concat(
+            "mv -f ",
+            _source,
+            " ",
+            _target
+        );
+        vm.ffi(updateGenesisOutFileCommands);
+    }
+
+    function _addAllocPropertyToGenesis(string memory _path) internal {
+        string memory tmpGenesisPath = string.concat(vm.projectRoot(), "/deployments/", deploymentContext, "/genesis.tmp.json");
+        string[] memory addAllocsKeyToGenesiscommands = new string[](3);
+        addAllocsKeyToGenesiscommands[0] = "bash";
+        addAllocsKeyToGenesiscommands[1] = "-c";
+        addAllocsKeyToGenesiscommands[2] = string.concat(
```

Comment:
Can you use the same subshell pattern as in the other PR to turn this into a 1 liner? ie no tempfile